packages/itwinui-css/src/input/input.scss

```
@@ -48,7 +48,7 @@
     cursor: not-allowed;
   }
 
-  @at-root :where(textarea#{&}) {
+  &:where(textarea) {
```

Specificity goes up with this. Should we have something like this to keep the old specificity?

```suggestion
  :where(textarea:is(&))
```

```css
/* Old */
:where(textarea.iui-input) {} /* 0,0,0 */
:where(textarea.iui-input)[disabled], :where(textarea.iui-input)[readonly] {} /* 0,1,0 each */

/* New */
.iui-input:where(textarea) {} /* 0,1,0 */
.iui-input:where(textarea)[disabled], .iui-input:where(textarea)[readonly] {} /* 0,2,0 each */
```

---

i think it doesn't really matter